apps/OboeTester/app/src/main/java/com/mobileer/oboetester/BaseAutoGlitchActivity.java

```diff
@@ -413,6 +414,8 @@ void testPerformancePaths() throws InterruptedException {
         StreamConfiguration requestedInConfig = mAudioInputTester.requestedConfiguration;
         StreamConfiguration requestedOutConfig = mAudioOutTester.requestedConfiguration;
 
+        boolean wasMMapEnabled = NativeEngine.isMMapEnabled();
```

Comment:
Shouldn't this can be simply cached by the following code?
```
final boolean requestedInConfigwasMmap = requestedInConfig.isMmap();
final boolean requestedInConfigwasMmap = requestedOutConfig.isMmap();
```

We don't need to know what is the actual mmap enabled or not. What we need is to keep the original requested configuration and request it after the test is completed.

Reply:
We don't need this because the only time we look at the requested configs are with testCurrentConfigurations(), which is within this function.

NativeEngine.isMMapEnabled() is good to keep the MMAP state unchanged after the test.

Reply:
That doesn't sound correct. For instance, if this test is first run, orignially, the global mmap enabled can be by default as `true`. The `requestedInConfig` can either have `isMmap` as true or false. It is also no necessary that the `inConfig` and the `outConfig` has the same `isMmap` value.

Reply:
Done! Thanks!